### ISSUE TITLE
File steps: p:copy-files and m

### ISSUE THREAD

**Issue by xml-project** [xproc/3.0-steps#354]
I propose an additional option on p:file-copy
<p:option name=„override“ as=„xs:boolean“ select=„false()“ />
Semantics: If a processor encounters a resource (file or folder) with the same relative URI in the target locations as in the source location AND override is false(), then the original content in target will be preserved (=no copy will take place). Equally no error is raised, the processor just moves on.
In other words: „override=true()“ creates a verbatim copy while „override=false()“ copies only missing files to the target.
Does this make sense?

**Comment by xatapult** [comment]
Seems like a useful addition to me. 👍

**Comment by xml-project** [comment]
Closed as #433 is more specific on this point.
